app/src/main/res/values-ru/strings.xml

```diff
@@ -70,5 +70,8 @@
     <string name="notification_service_pokemon_in_area">%1$d покемон в досягаемости:</string>
     <string name="meters">метров</string>
     <string name="toast_google_login_error">Ошибка входа через аккаунт Google</string>
-
+    <string name="toast_still_searching">Поиск&#8230; найден %1$d Pokémon поблизости.</string>
```

Comment:
we could write a pluralizer for that... but for the time being, your solution is OK